packages/react-components/react-toolbar/src/stories/ToolbarWithPopover.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Toolbar, ToolbarProps } from '../index';
+import { ToolbarButton } from '../ToolbarButton';
+import { ToolbarDivider } from '../ToolbarDivider';
+import { Popover, PopoverTrigger, PopoverSurface } from '@fluentui/react-popover';
+import { CalendarMonthRegular } from '@fluentui/react-icons';
+
+export const WithPopover = (props: Partial<ToolbarProps>) => (
+  <Toolbar {...props} size="small">
+    <Popover>
+      <PopoverTrigger>
+        <ToolbarButton>Click here</ToolbarButton>
+      </PopoverTrigger>
+      <PopoverSurface>
+        <div>Popover content</div>
+      </PopoverSurface>
+    </Popover>
```

Comment:
nit: I would extract this and make a `ToolbarButtonWithPopover` which is more realistic of what would happen in real usage

Reply:
You mean exporting `ToolbarButtonWithPopover` as a component part of `Toolbar`?

Reply:
no, just as a part of the story